swagger_spec_validator/validator20.py

```diff
@@ -454,22 +455,24 @@ def validate_arrays_in_definition(definition_spec, deref, def_name=None, visited
             definition=definition_spec['items'],
             deref=deref,
             def_name='{}/items'.format(def_name),
-            visited_definitions_ids=visited_definitions_ids,
+            visited_definitions=visited_definitions,
         )
 
 
-def validate_definition(definition, deref, def_name=None, visited_definitions_ids=None):
+def validate_definition(definition, deref, def_name=None, visited_definitions=None):
     """
-    :param visited_definitions_ids: set of ids of already visited definitions (after dereference)
+    :param visited_definitions: set of ids of already visited definitions (after dereference)
```

Comment:
This docstring needs to be updated since this is now used differently.

Reply:
Fixed the docstring